app/models/report.rb

```diff
@@ -121,34 +123,37 @@ def self.compressed_report?
   end
 
   def normal_report?
-    return nil if self.compressed_report? || self.resolution_report?
+    return nil if compressed_report? || resolution_report?
+
     true
   end
 
   def compressed_report?
-    return nil if self.exceptions.nil? || self.exceptions.empty?
+    return nil if exceptions.nil? || exceptions.empty?
```

Comment:
you can achieve this check by using the Rails blank? method.
`return nil if exceptions.nil? || exceptions.empty?`
becomes
`return nil if exceptions.blank?`

Reply:
Auto format changed whitespace. I did not change that code.

The only thing I;ve changed is ASC to DESC

I don't want to change anything else in this PR

Reply:
@richardhallett understood